### PR TITLE
gpgstats: init at 0.5

### PR DESCRIPTION
**Pull request by davidak**
it compiles and works on NixOS x86_64.

### CODE REVIEW

File: pkgs/tools/security/gpgstats/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  buildInputs = [ ncurses gpgme ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp gpgstats $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "calculates statistics on the keys in your gpg key-ring";
```
Comment (jagajaga) [inline pkgs/tools/security/gpgstats/default.nix]: Start with a capital letter please.

File: pkgs/tools/security/gpgstats/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    longDescription = ''
+    GPGstats calculates statistics on the keys in your key-ring.
```
Comment (jagajaga) [inline pkgs/tools/security/gpgstats/default.nix]: Remove usage please :)

**Comment by davidak** [comment]
@jagajaga requested changes applied. still builds.

**Comment by domenkozar** [comment]
it doesn't build on 32bit: http://hydra.nixos.org/build/27422407/nixlog/3/raw

**Pull request merged**